README.md

```diff
@@ -91,9 +88,30 @@ Comprehensive documentation for QuEmb is available at `quemb/docs`. The document
 
 Alternatively, you can view the latest documentation online [here](https://quemb.readthedocs.io/).
 
-## References
+# References
 
-The methods implemented in this code are described in details in the following papers:
+### For this library (QuEmb):
+- [INSERT SOFTWARE PAPER HERE]
+
+### For general BE theory:
+- Welborn, M., Tsuchimochi, T. & Van Voorhis, T. Bootstrap embedding: An internally consistent fragment-based method. The Journal of Chemical Physics 145, 074102 (2016). ()
+- Ye, H.-Z., Ricke, N. D., Tran, H. K. & Van Voorhis, T. Bootstrap Embedding for Molecules. J. Chem. Theory Comput. 15, 4497–4506 (2019).
+- Ye, H.-Z., Tran, H. K. & Van Voorhis, T. Bootstrap Embedding For Large Molecular Systems. J. Chem. Theory Comput. 16, 5035–5046 (2020).
+- Ye, H.-Z. & Van Voorhis, T. Atom-Based Bootstrap Embedding For Molecules. J. Phys. Chem. Lett. 10, 6368–6374 (2019).
+
+### For periodic BE theory and implementation: 
 - OR Meitei, T Van Voorhis, Periodic bootstrap embedding, [JCTC 19 3123 2023](https://doi.org/10.1021/acs.jctc.3c00069)  
 - OR Meitei, T Van Voorhis, Electron correlation in 2D periodic systems, [arXiv:2308.06185](https://arxiv.org/abs/2308.06185)  
-- HZ Ye, HK Tran, T Van Voorhis, Bootstrap embedding for large molecular systems, [JCTC 16 5035 2020](https://doi.org/10.1021/acs.jctc.0c00438)
+
+### For BE-DMRG theory and implementation:
+- [INSERT BE-DMRG PAPER HERE]
+
+### For Unrestricted BE theory and implementation:
+- Tran, H. K., Ye, H.-Z. & Van Voorhis, T. Bootstrap embedding with an unrestricted mean-field bath. The Journal of Chemical Physics 153, 214101 (2020).
+- [INSERT LEAH PAPER HERE]
+
```

Comment:
I think (yet) unpublished papers can be added once they can be cited?

Reply:
Indeed, these are placeholders that will ensure proper attributions are made in the future.

Reply:
This should probably come as a completely separate PR! List of BE papers can be included in the documentation  but README is probably not the best place. The references here are for QuEmb library.

Reply:
That's a fair point, I'll open a few new PRs in the coming days with all of this reorganized.